.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Deriv Analytics Test Workflow
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
isn't this ran the workflow twice unnecessarily ?

Reply:
i'm also running it on master in case that merge causes any issue we notice it. 
the other reason is that I'm saving the cache on merge to master. so its needed